boards/nordic/nrf54l15pdk/nrf54l15pdk_nrf54l15_cpuapp_0_2_1.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
```

Comment:
After discussion with @barsok, I desist from adding support for this board variant to twister (not needed).